### PR TITLE
Dashing

### PR DESCRIPTION
**Pull request by LanderU**

### CODE REVIEW

**Comment by LanderU** [comment]
Ready to be approved and merged!

File: mara-ros2.repos [hunk 1]
```diff
@@ -2,20 +2,36 @@ repositories:
...
+      url: https://github.com/ros-simulation/gazebo_ros_pkgs
+      version: ros2
+  control_msgs:
+    type: git
+    url: https://github.com/erlerobot/control_msgs/
+    version: crystal
+  geometry2:
+    type: git
+    url: https://github.com/acutronicrobotics/geometry2/
+    version: ros2
```
Comment (YueErro) [inline mara-ros2.repos]: I have had to remove geometry2 repository in order to be able to compile it today. Everything else has worked fine.
I have dashing-desktop installed.

Comment (rkojcev) [reply mara-ros2.repos]: need to test this as well. I plan to move to dashing today and see which repos are working

**Comment by rkojcev** [comment]
We should make a release for crystal and merge dashing in the master. Lets do one last sweep of repo today and tag master for crystal.

**Comment by ahcorde** [comment]
Can we speed up this?  We need this for HRIM

There is an issue with the size of the gripper. Can we fix this problem?

**Comment by YueErro** [comment]
I guess is fine now.

**Pull request merged**